### DISCUSSION TITLE
Prevent using images from phone gallery as a song artwork

### DISCUSSION THREAD

**Discussion by mmdoff1** [namidaco/namida#338] [Ideas]
• Hi I just saw this app on social media and wanted to give it a try. While browsing around I saw that some of the songs in the app uses my taken (personal) pictures as a song artwork and this is really annoying imo.

• (I never changed the artwork or never added one before, there is not any image on the title when I check it from my device's storeage)

• Is there any way to disable this? If not could this feature be added in future updates please? Thanks in advance!

**Comment by MSOB7YY** [comment]
hey the image picked is not random, it should be a fallback image that exists in the same folder of the track
usually when the track has no artwork embedded, it tries to look in the folder for image files, the priority is for files like "artwork.jpg" "folder.png" etc, otherwise it picks any image in that folder. 
can you confirm that the image displayed **actually** exists in the track folder? can you send me the path of the 2 files?

**Reply by mmdoff1** [reply]
Hey thanks for your reply! No I do not have any images added to that folder on my device's storage. I mostly download songs (.mp3) from Telegram, that's why the folder named as "Telegram" and it doesn't have any image on it's thumbnail. I tried to check and edit the folder but unfortunately I am unable to do so because there is nothing as I said. The app still countinues to show the same image.

**Reply by MSOB7YY** [reply]
can you provide me with both the music file path and the image file path?

**Reply by MSOB7YY** [reply]
smth like this: `"/storage/emulated/0/Music/10 After The Rain (feat. Such).flac"`

from any filemanager (mixplorer for example), u select the file and show the details

**Reply by mmdoff1** [reply]
Okay so, the song folder is in: "My phone-> Internal storage->Music->Telegram.

The image that the app uses is in: "My phone->Internal storage->Download->Telegram"

**Reply by MSOB7YY** [reply]
hey i checked and there was a silly bug that caused this. i also made it more restricted so it should be fine. u can try v4.2.0 beta from here https://github.com/namidaco/namida-snapshots/releases or wait the next stable version, thanks for reporting tho

**Reply by mmdoff1** [reply]
Thank you very much for checking for it! I really appreciate that 🙏 

You and your app are GREAT. Keep going. ❤️🤗

**Reply by MSOB7YY** [reply]
its all thanks to you for making me notice it.. much luvv 💗💗
